src/Rules/PropertyConditions/ElementGroups.cs

```diff
@@ -169,14 +168,16 @@ private static Condition CreateNonFocusableSliderButtonsCondition()
 
         private static bool IsParentWPFDataItem(IA11yElement e)
         {
+
+            // GetUIFramework searches the ancestors for a valid framework value in addition to checking the element itself
             return e.GetUIFramework() == Core.Enums.FrameworkId.WPF
                 && e.Parent != null
                 && e.Parent.ControlTypeId == Axe.Windows.Core.Types.ControlType.UIA_DataItemControlTypeId;
         }
 
         private static bool HasAllowedPlatformPropertiesForText(IA11yElement e)
         {
-            return !IsPlatformWinForms(e)
+            return !WinForms.Matches(e)
```

Comment:
Are we concerned about e ever being null at this point? The old code would have simply returned false due to the null propagation. I think we'll get a NullReferenceException if a null makes it to here, and I'm not finding any guard against it

Reply:
Eventually this code will make it to the extension method GetUIFramework, which has a null check in it. When that happens, null is returned for the value of the string. So this should not throw an exception if e is null